### PR TITLE
Add support for --log-level

### PR DESCRIPTION
**Pull request by ndeloof**
**What I did**
Added support for `--log-level` for consistency with docker CLI

**Related issue**
fixes https://github.com/docker/compose-cli/issues/1213

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**
image

### CODE REVIEW

File: cli/main.go [hunk 1]
```diff
@@ -146,20 +146,20 @@ func main() {
...
-	root.PersistentFlags().StringVarP(&opts.LogLevel, "log-level", "l", "info", "Set the logging level (\"debug\"|\"info\"|\"warn\"|\"error\"|\"fatal\")")
-	root.PersistentFlags().BoolVarP(&opts.Debug, "debug", "D", false, "Enable debug output in the logs")
-	root.PersistentFlags().StringVarP(&opts.Host, "host", "H", "", "Daemon socket(s) to connect to")
-	opts.AddContextFlags(root.PersistentFlags())
-	opts.AddConfigFlags(root.PersistentFlags())
-	root.Flags().BoolVarP(&opts.Version, "version", "v", false, "Print version information and quit")
+	flags := root.Flags()
+	flags.StringVarP(&opts.LogLevel, "log-level", "l", "info", "Set the logging level (\"debug\"|\"info\"|\"warn\"|\"error\"|\"fatal\")")
+	flags.BoolVarP(&opts.Debug, "debug", "D", false, "Enable debug output in the logs")
+	flags.StringVarP(&opts.Host, "host", "H", "", "Daemon socket(s) to connect to")
```
Comment (gtardif) [inline cli/main.go:152]: Just confirming, these are not set as persistent flags in the Docker CLI, OK

Comment (gtardif) [reply cli/main.go:152]: Actually help is a persistent flag (and that probably is the cause for red tests)

Comment (gtardif) [reply cli/main.go:152]: So we need to keep help as persistent in compose-cli

Comment (ndeloof) [reply cli/main.go:152]: -H is to set DOCKER_HOST, not help

--help is registered a few line later on each sub command:
```
walk(root, func(c *cobra.Command) {
		c.Flags().BoolP("help", "h", false, "Help for "+c.Name())
	})
```

Comment (gtardif) [reply cli/main.go:156]: 🤦 of course...

**Review by gtardif — APPROVED** [review]

**Pull request merged**